patches/0001-autotools-m4-manage-MODULE_IMPORT_NS-prototype-chang.patch

```diff
@@ -9,21 +9,23 @@ EXPORT_SYMBOL_NS_GPL double-quoting in generated .mod.c files.
 Uses feature detection instead of macro overrides to avoid interfering
```

Comment:
Last 3 commits are just rebase. please merge and make it one commit. no need of separate